apm/apm.go

```diff
@@ -222,9 +222,10 @@ func (a *APM) joinSubnet(fullName string) error {
 	}
 
 	subnet := definition.Definition
+	subnetID, _ := subnet.GetID(constant.DefaultNetwork)
```

Comment:
@dasconnor All the subnet definitions we have currently are only in testnet, so I've hard-coded this to only use this for now. I can expose this as a `--network` flag in the next release (or I can do this for this release if we feel this is important)

Reply:
This is good for now. We're testnet only for the moment, so we probably won't mess with this until a sprint or two down the road.